xml/docker_dockerizing_application.xml

```diff
@@ -225,62 +225,123 @@ ENTRYPOINT ["apache2ctl"]</screen>
+           <para>Build the container. Set a tag for your image with the <option>-t</option>
+            option (here <literal>&exampleuser_plain;/apache2</literal>, but
+           you can use any name you want):</para>
+           <screen>sudo docker build -t &exampleuser_plain;/apache2 .</screen>
+           <para>Docker refreshes repositories and installs the Apache2
+            server as it is not installed by default in the &slsa; Docker image.</para>
+          </step>
+          <step>
+           <para>Run the image in <quote>detached</quote> mode:</para>
+           <screen>docker run --detach --interactive --tty &exampleuser_plain;/apache2</screen>
+           <para>Docker responds with the container ID, for example:</para>
+           <screen>7bd674eb196d330d50f8a3cfc2bc61a243a4a535390767250b11a7886134ab93</screen>
+          </step>
+          <step>
+           <para>Open a Web browser and enter in the text field <uri>http://localhost:80/test.html</uri>.
+           You should see the output <literal>The Web Server is running</literal>.
+           </para>
+          </step>
+         </procedure>
+         <para>With the last procedure, you have built an image which was used
+         to test the build process and the Apache2 Web server. If everything
+         was successful, you can store your own data. Proceed as follows:</para>
+         <procedure>
+          <title>Creating a Docker Container with your Own Data</title>
+          <step>
+           <para>Stop and remove the previous container with:</para>
+           <screen>docker ps --latest
+CONTAINER ID        IMAGE               COMMAND                  [...]
+<emphasis role="bold">afee0124a0c7</emphasis>        tux/apache2         "/usr/sbin/httpd -..."   [...]
+docker rm --force afee0124a0c7</screen>
+          </step>
+          <step>
+           <para>Remove or disable the echo line <xref linkend="co.dockerfile.apache2.echo"/>
+            in <xref linkend="ex.docker.dockerfile.apache2"/> and remove the
+            hash character in the first column of the copy line
+           <xref linkend="co.dockerfile.apache2.copy"/>.
+           </para>
+          </step>
+          <step>
+           <para>Rebuild the image as described in <xref linkend="st.docker.apache2.build"/>
+            of <xref linkend="pro.docker.apache2" xrefstyle="select:label"/>.</para>
+          </step>
+          <step>
+           <para>Run the image in <quote>detached</quote> mode:</para>
+           <screen>docker run --detach --interactive --tty &exampleuser_plain;/apache2</screen>
+           <para>Docker responds with the container ID, for example:</para>
+           <screen>e43fff4ae9832ecdb7677c058a73039d7610c32145a1d9b6ad0a4ed52b5c4dc7</screen>
+          </step>
+         </procedure>
+         <para>You can now access your data in <uri>http://localhost:80</uri>.
+         If you do not want to copy your data into your Docker container,
```

Comment:
replace 'your' with 'the'. i think anything with 'you' is used too often.

Reply:
@tomschr Please use \&docker;